jdk/make/closed/autoconf/custom-spec.gmk.in

```diff
@@ -68,9 +68,8 @@ USERNAME                := @USERNAME@
 
 include $(SRC_ROOT)/closed/openjdk-tag.gmk
 
-JDK_MOD_VERSION         := @JDK_MOD_VERSION@
 JDK_FIX_VERSION         := @JDK_FIX_VERSION@
-J9JDK_EXT_VERSION       := $(JDK_MINOR_VERSION).$(JDK_MICRO_VERSION).$(JDK_UPDATE_VERSION).$(COOKED_BUILD_NUMBER)-ea
+J9JDK_EXT_VERSION       := $(JDK_MINOR_VERSION).$(JDK_MICRO_VERSION).$(JDK_UPDATE_VERSION).$(JDK_FIX_VERSION)-ea
```

Comment:
I think `openj9ext-version-numbers` should set `JDK_FIX_VERSION=0` so developer builds will continue to say `...0-ea` instead of `...HEAD-ea` like this:
```
$ java -Xinternalversion
Eclipse OpenJ9 OpenJDK 64-bit Server VM (1.8.0_352-internal-keithc_2022_08_17_09_52-b00) from linux-amd64 JRE with Extensions for OpenJDK for Eclipse OpenJ9 8.0.352.HEAD-ea, built on Aug 17 2022 09:56:38 by keithc with g++ (Ubuntu 7.5.0-3ubuntu1~18.04) 7.5.0
```

Reply:
That works for me, I've made the change.